sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1446,7 +1446,7 @@ object SQLConf {
       " register class names for which data source V2 write paths are disabled. Writes from these" +
       " sources will fall back to the V1 sources.")
     .stringConf
-    .createWithDefault("")
+    .createWithDefault("orc")
```

Comment:
@gengliangwang @cloud-fan Please discuss it with @rdblue and see whether we can keep using data source v2 API even if the table catalog is not ready.

Reply:
This is required. We cannot move sources to v2 by default until we are confident that they will behave the same way. We can't do that until v2 is actually implemented.